pyaedt/icepak.py

```diff
@@ -1540,18 +1549,17 @@ def create_temp_point_monitor(self, point_name, point_coord = [0,0,0]):
 
     @aedt_exception_handler
     def delete_em_losses(self, bound_name):
-        """Delete EM losses boundaries.
+        """Delete EM losses boundary.
 
         Parameters
         ----------
         bound_name : str
-            Name of the boundary to delete
```

Comment:
Personally, I think "to delete" is fine here, but that's totally up to you as both work.